### PR TITLE
[DEGR-1917] Add py310 as option for runtime in Cognite Functions

### PR DESCRIPTION
**Pull request by qTipTip**
## Description
Updates a docstring to reflect the addition of Python 3.10 as supported runtime in Cognite Functions. 

## Checklist:
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.

### CODE REVIEW

**Review by erlendvollset — CHANGES_REQUESTED** [review]
No need to bump version and update changelog for docs changes

**Comment by qTipTip** [comment]
> No need to bump version and update changelog for docs changes

Aight, thanks!

**Review by erlendvollset — APPROVED** [review]

File: cognite/client/_api/functions.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def create(
...
             cpu (Number, optional):                  Number of CPU cores per function. Allowed values are in the range [0.1, 0.6], and None translates to the API default which is 0.25 in GCP. The argument is unavailable in Azure.
             memory (Number, optional):               Memory per function measured in GB. Allowed values are in the range [0.1, 2.5], and None translates to the API default which is 1 GB in GCP. The argument is unavailable in Azure.
-            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", `None`], and `None` translates to the API default which currently is "py38". The runtime "py38" resolves to the latest version of the Python 3.8 series.
+            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", "py310", `None`], and `None` translates to the API default which currently is "py38". The runtime "py38" resolves to the latest version of the Python 3.8 series.
```
Comment (haakonvt) [inline cognite/client/_api/functions.py:125]: Just a suggestion:
```suggestion
            runtime (str, optional):                 The function runtime. Valid values are ["py37", "py38", "py39", "py310", `None`], and `None` translates to the API default which will change over time. The runtime "py38" resolves to the latest version of the Python 3.8 series.
```
I also think you should update the official API docs before doing the Python SDK:

image

Comment (qTipTip) [reply cognite/client/_api/functions.py:125]: Thanks!

I've updated it here: https://github.com/cognitedata/doc.cognitedata.com/pull/1697

Comment (qTipTip) [reply cognite/client/_api/functions.py]: Ah, the API docs.
I'll fix before merging.

Comment (qTipTip) [reply cognite/client/_api/functions.py]: https://github.com/cognitedata/service-contracts/pull/2201

Comment (haakonvt) [reply cognite/client/_api/functions.py]: # 🚀🚀

**Pull request merged**